### ISSUE TITLE
feat[dev]: ✨ implement A11y improvements for Media Template

### ISSUE THREAD

**Issue by yoezlem** [solid-design-system/solid#1533]
## Description
This task describes the Storybook implementation for the Media template as part of the A11y improvements outlined in Epic https://github.com/solid-design-system/solid/issues/1465.

A11y improvements for the Media template can be found here: [Media Notes](https://www.notion.so/matuzo/Media-112f3f6e1bc0807fa259dc48b34ceae0).

------------

## Comment
**Problem:** The copyright text is hard to read. Tools don’t/can’t provide a ratio when testing against background images, but technically, it’s a violation of [WCAG 1.4.3 Contrast (Minimum)](https://www.w3.org/WAI/WCAG22/quickref/?versions=2.1&showtechniques=143#contrast-minimum).
**Solution:** Consider using background colors for copyright text.

## Related WCAG criteria
- https://www.w3.org/WAI/WCAG22/quickref/?versions=2.1&showtechniques=143#contrast-minimum
- https://www.w3.org/WAI/WCAG22/Techniques/failures/F83.html

## DoR
- [ ] Item has been estimated by the team
- [x] Item dependencies have been identified and documented

## DoD
- [ ] Documentation has been created/updated (if applicable)
- [ ] Migration Guide has been created/updated (if applicable)
- [ ] Relevant E2E tests (Features, A11y, Bug fixes) are created/updated
- [ ] Relevant stories (Features, A11y) are created/updated
- [ ] Implementation works successfully on `feature` branch

**Comment by yoezlem** [comment]
will be closed after: #1309
